prow/pjutil/filter.go

```diff
@@ -40,10 +41,15 @@ var OkToTestRe = regexp.MustCompile(`(?m)^/ok-to-test\s*$`)
 // 1. presubmits that can be run with '/test all' command.
 // 2. optional presubmits commands that can be run with their trigger, e.g. '/test job'
 // 3. required presubmits commands that can be run with their trigger, e.g. '/test job'
-func AvailablePresubmits(changes config.ChangedFilesProvider, org, repo, branch string, presubmits []config.Presubmit, logger *logrus.Entry) ([]string, []string, []string, error) {
+func AvailablePresubmits(changes config.ChangedFilesProvider, org, repo, branch string,
+	presubmits []config.Presubmit, logger *logrus.Entry) (prowflagutil.Strings, prowflagutil.Strings, prowflagutil.Strings, error) {
```

Comment:
Why is this returning flags?

Reply:
I searched for a strings set wrapper in the source code and found `Strings` under `flagutil`. A set to support unique values.
I will replace to use `StringsSet` directly instead.